_minutes/2024-09-27-wecg-tpac.md

```diff
@@ -0,0 +1,683 @@
+   * [rob] Should we expose in an event?
+   * [devlin] Would make sense. Should table for now.
+   * **Resolution**: Close
+   * devlin to add comment and close out
+ * [Issue 639](https://github.com/w3c/webextensions/issues/639): i18n.getMessage() pluralization
+   * [devlin] Solved with message format 2, right?
+   * [timothy] Agreed.
+   * [devlin] Even messages TC39 proposal for Intl.MessageFormat.
+   * [oliver] Close in favor of issue 698?
+   * [devlin] Yes.
+   * **Resolution**: Close, the work will be tracked by [issue 698](https://github.com/w3c/webextensions/issues/698).
+ * [Issue 640](https://github.com/w3c/webextensions/issues/640): Proposal: Remove multiple bookmarks in bookmarks.remove()
+   * [devlin] No objections. Not fond of options (e.g. string | strings[]) but makes sense in this context. Performance benefits and race condition benefits to making this a single operation.
+   * [rob] What would the race condition be?
+   * [devlin] If it's an atomic operation you can be assured that you will remove all of them at once.
+   * [rob] Would we specify this to be atomic? Either all or nothing.
+   * [devlin] That's not a part of triage ;) Good question. Inclined to say we'll do our best and it's okay to have partial removal. If you ask to remove 50 and we only remove 49 because one was already removed, that's acceptable.
+   * [timothy] We don't support bookmarks yet, but this would be preferable for us as we use a DB for bookmarks.
+   * **Resolution**: Everyone supportive of the ability to remove multiple bookmarks at once.
+ * [Issue 615](https://github.com/w3c/webextensions/issues/615): userstyles support
+   * [rob] TL;DR is request to add support for registering CSS in the scripting/userScripts API.
+   * [timothy] duplicating something already in scripting...
+   * [devlin] So we have file support for registering stylesheets in scripting.registerContentScripts, but not arbitrary css code. Supportive of adding this, but through the scripting API. We do not need to add it to the userScripts namespace since the scripting namespace already accepts arbitrary CSS code in scripting.insertCSS.
+   * [rob] So to recap, we are all supportive of supporting arbitrary stylesheets through scripting.registerContentScripts, but not in the userScripts API.
+   * [devlin] Correct.
+   * [timothy] Yes.
+   * **Resolution:** Add support for registering user styles to the `scripting.registerContentScripts()` method.
+ * [Issue 624](https://github.com/w3c/webextensions/issues/624): Proposal: getLeafTarget() method
+   * [oliver] When discussed, both Firefox and Safari were supportive. I chatted with Devlin, and wonder whether this could be added to the web side, with an extension-only option to opt in to closed shadow roots. I spoke to one of the Dominics on the Chrome side, and there is strong opposition to adding this to the web platform. Someone from Apple had similar reservations.
```

Comment:
```suggestion
   * [oliver] When discussed, both Firefox and Safari were supportive. I chatted with Devlin, and wonder whether this could be added to the web side, with an extension-only option to opt in to closed shadow roots. I spoke to one of the Dominics on the Chrome side, and there is an expectation that we would encounter strong opposition to adding this to the web platform. Someone from Apple had similar reservations.
```